code/modules/mob/living/carbon/human/appearance.dm

```diff
@@ -211,4 +209,4 @@
 /mob/living/carbon/human/proc/force_update_limbs()
 	for(var/obj/item/organ/external/O in organs)
 		O.sync_colour_to_human(src)
-	update_body(0)
+	update_icons_body()

```

Comment:
I think this can be update_icons_body(FALSE)  since the old one didn't call update_icons either.